packages/shopping/public/ui.js

```diff
@@ -232,17 +211,15 @@ async function displayShoppingCart() {
   }
 }
 
-function updateCount(productId, quantity) {
-  api.getShoppingCartItems(function(result) {
-    result.items.forEach(item => {
-      if (item.productId === productId) {
-        item.quantity = +quantity;
-      }
-    });
-    api.updateCart(result.items, function() {
-      updatePrice(quantity);
-    });
+async function updateCount(productId, quantity) {
```

Comment:
Why isn't this:

```
updateCount({productId, quantity})
```

like the other ones you change to be ` ( { parm1,...parmN} ) `  ?

Reply:
Good suggestion.